tests/test_concurrency.py

```
@@ -34,4 +36,32 @@ def test_concurrent_writes():
     assert len(set(versions)) == 1000
 
     res = common.g.query("MATCH (n:Node) RETURN count(DISTINCT n.id)")
-    assert res.result_set[0][0] == 1000
\ No newline at end of file
+    assert res.result_set[0][0] == 1000
+
+def run_write_burst(id):
```

Ruff (0.15.20)

[error] Function argument `id` is shadowing a Python builtin

(A002)